codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -25,16 +26,30 @@
 
     private final String clientIdentifier;
     private final String willTopic;
-    private final String willMessage;
+    private final byte[] willMessage;
     private final String userName;
-    private final String password;
+    private final byte[] password;
 
     public MqttConnectPayload(
```

Comment:
should we deprecate this constructor in favor of the new constructor which takes `byte[]`?